DNN Platform/Admin Modules/Dnn.Modules.Console/web.config

```diff
@@ -1,11 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-	<runtime>
+  <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30AD4FE6B2A6AEED" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
       </dependentAssembly>
     </assemblyBinding>
-	</runtime>
-</configuration>
+  </runtime>
+  <system.web>
```

Comment:
if this gets deployed, I think IIS will complain because it's not allowed to have this element declared except in the root `web.config`

Reply:
I wasn't 100% sure about this, I can go and remove it. Thanks for the feedback